app/src/main/java/com/udacity/bakingapp/fragment/RecipeStepFragment.java

```diff
@@ -80,8 +83,8 @@ public void onActivityCreated(@Nullable Bundle savedInstanceState) {
         super.onActivityCreated(savedInstanceState);
         Bundle arguments = getArguments();
 
-        final String description = arguments.getString(EXTRA_DESCRIPTION);
-        final String mediaURL = arguments.getString(EXTRA_MEDIA_URL);
+        description = arguments.getString(EXTRA_DESCRIPTION);
```

Comment:
Looks like description is saved but isn't used as part of restoration.

Reply:
That's for completeness, only the exoplayer requires being explicitly restored since it's on a separate `ExoPlayerImpl` thread that gets released during onPause/onStop